src/SonarScanner.MSBuild/BootstrapperClass.cs

```diff
@@ -91,6 +86,15 @@ public async Task<int> Execute()
             return exitCode;
         }
 
+#if NETFRAMEWORK || NETCOREAPP2_1
+
+        protected virtual void WarnAboutDeprecation(ITeamBuildSettings teamBuildSettings)
```

Comment:
I have an itch on this (I think it's fine to be a no-op virtual method for other frameworks), but I also like the fact that by doing this:
- it won't appear outside of the scope
- it will be easy to find and remove

Reply:
Honestly, I wanted to somehow "mark it" so when everything else related to the warnings gets removed, this is not missed. It is not the nicest thing, however the whole solution is a bit itchy for me, not just this part 😄